src/components/VideoDetailsCard/VideoDetailsCard.component.js

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import CurrentVideoContext from '../../context/current-video-context';
+import {
+  VideoDetailsIframe,
+  VideoDetailsCardStyled,
+} from './VideoDetailsCard.styled';
+
+const VideoDetailsCard = () => {
+  const currentVideoContext = useContext(CurrentVideoContext);
+  return (
+    <VideoDetailsCardStyled>
+      <VideoDetailsIframe
+        src={`https://www.youtube.com/embed/${currentVideoContext.videoDetails.id}`}
```

Comment:
will be a good practice to destructure all the videoDetails properties

Reply:
Done! ⭐